client/src/pages/Admin.js

```diff
@@ -7,51 +8,73 @@ import React from "react";
 import { useState, useEffect } from "react";
 
 function AdminPage() {
-  const [pagesData, setPagesData] = useState([]);
+	const [pagesData, setPagesData] = useState([]);
 
-  async function fectPageTitles() {
-    try {
-      const getPageTitles = await fetch("/api/pages");
-      const allPageTitles = await getPageTitles.json();
-      return allPageTitles;
-    } catch (error) {
-      console.error(error);
-      return "Page TITLES  are not coming!! investigate why";
-    }
-  }
+	async function fectPageTitles() {
+		try {
+			const getPageTitles = await fetch("/api/pages");
+			const allPageTitles = await getPageTitles.json();
+			return allPageTitles;
+		} catch (error) {
+			console.error(error);
+			return "Page TITLES  are not coming!! investigate why";
+		}
+	}
 
-  async function fectPagesData(pageTitle) {
-    try {
-      const getPagesData = await fetch(`api/pages/${pageTitle}`);
-      const pagesData = await getPagesData.json();
-      return pagesData;
-    } catch (error) {
-      console.error(error);
-      return "Pages DATA are not  coming!! investigate why";
-    }
-  }
+	async function fectPagesData(pageTitle) {
+		try {
+			const getPagesData = await fetch(`api/pages/${pageTitle}`);
+			const pagesData = await getPagesData.json();
+			return pagesData;
+		} catch (error) {
+			console.error(error);
+			return "Pages DATA are not  coming!! investigate why";
+		}
+	}
+
+
+   async function handleModuleDelete(pageTitle, recordId) {
+			const confirmed = confirm("Are you sure you want to delete?");
+
+			if (confirmed) {
+				try {
+          const response = await fetch(`api/pages/${pageTitle}/${recordId}`, {
+            method: "Delete",
+          });
+					const deletingModule = await response.json();
+					console.log(deletingModule);
```

Comment:
We don't need console.log
console.error is fine but console.log is for development stage

Reply:
Done